modules/nw-config-ip-failover.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
```

Comment:
We don't sopport IP failover in 4.1/4.2, this file `nw-config-ip-failover.adoc` can be removed.